Makefile.am

```diff
@@ -325,6 +325,23 @@ container-rpms:
 container-rpms-scratch:
 	$(MAKE) -f ./Makefile.am CONTAINER_ADD_ARGS="-e TEST_BUILD=true" container-rpms
 
+# Build ISO from Anaconda RPM files build here
+container-iso-build-scratch:
+	@echo "To build an ISO we need to have root privilidges because of loop device mounting in Lorax!"
+	@echo "You need to have anaconda RPMs build first. Please run 'make -f ./Makefile.am container-rpms-scratch' before this command."
```

Comment:
Why don't you actually test for this? I'd imagine something like `if ! sudo -v ; then echo "..."; exit 1; fi` and for the RPMs something like `if compgen -E "./result/build/01-rpm-build/*.rpm" > /dev/null ; then echo "..."; exit 1; fi`.

Reply:
Indeed, that looks like it should not be that hard to do & could make the whole thing much more user friendly. :)

Reply:
I can definitely add that but not sure if this makes sense to complicate just the `echo` call?

If you insist on this I'm fine with adding the logic but right now it's seems to me just as more complication (lower readability) for not a big benefit.

Reply:
I added the checks.

Reply:
Thank you, but I would like also the `exit 1` part, so that we don't have to hunt for these lines on top of error messages from lorax in podman. There are clear prerequisites, you can technically check for them, so why not do it? Just telling the user there will be errors is pointless if you can also stop doing the thing that won't work.

Reply:
I'll add `exit 1` for missing RPMs but it doesn't give sense for the `We will ask for password` message.